files-community/Files
# Code Quality: Refactored ToolbarViewModel

## Summary

Refactored ToolbarViewModel and separated it into ToolbarViewModel and PathBreadcrumbViewModel.

## Task Checklist

- [x] Refactored ToolbarViewModel
- [x] Add PathBreadcrumbViewModel
- [ ] Move code from ToolbarViewModel

## Known Issues

_N/A_

## PR Checklist


## Steps To Validate Changes

1. Made window small and confirmed the search box visual states are working correctly, clicking the small search button changes the path bar into a search box.
2. Confirmed the command palette can be toggled by typing `>` and using the keyboard shortcut.
3. Confirmed the styling hasn't changed
4. Confirmed the breadcrumb drop downs work as expected

## Screenshots

_N/A_

[^1]: If the request hasn't been agreed by the team, this work might be rejected. Make sure that you get approval from the team before opening any PR related the request.
[^2]: If you removed any en-US string resources, make sure that there are no references of those resources. When you add a new en-US string resources, do not make any changes on other languages' resources; [Crowdin](https://crowdin.com/project/files-app) will do that, instead.
